file_parser.py

```diff
@@ -0,0 +1,135 @@
+    Supported extensions: pdf, odt, docx, txt
+    """
+    if file_path.split(".")[-1] != "txt":
+        text = textract.process(file_path).decode('utf8')
+    else:
+        with open(file_path, "rb") as f:
+            text = f.read().decode('utf8')
+    return text
+
+def clear_text(text: str) -> str:
+    """ 
+    Clears the text from punctuation by replacing every character 
+    specified in the regrex with a space .
+    """
+    return re.sub('[^A-Za-z0-9ĘęÓóĄąŚśŁłŻżŹźĆćŃń ]+', ' ', text)
+
+def recognize_data_strings(text: str, language: Language) -> list:
+    """
+    Returns list of strings containing data recognized as important
+    such as dates, organization names, people's names and surnames and others.
+    """
+    # TODO We might want to return not only the phrases but the categories as well
+    important_phrases = []
+    if language == Language.ENGLISH:
+        model = "en_core_web_lg"
+    elif language == Language.POLISH:
+        model = "pl_core_news_lg"
+    nlp = spacy.load(model)
+    important_text = nlp(text)
+    for ent in important_text.ents:
+        important_phrases.append(ent.text)
+    return important_phrases
+
+def lemmatize(words: List[str], language: Language) -> List[str]:
+    """
+    Creates a list of lemmatized words based on provided list of strings (words). 
+    """
+    if language == Language.ENGLISH:
+        lemmatizer = nltk.WordNetLemmatizer()
+        lemmatized_words = [lemmatizer.lemmatize(word) for word in words]
+        return lemmatized_words
+    elif language == Language.POLISH:
+        morf = morfeusz2.Morfeusz() 
+        last_i = -1
+        lemmatized_words = []
+        for i, _, interp in morf.analyse(' '.join(words)):
+            # using last_i number to only save the first interpretation of a word
+            if last_i < i:
+                lemmatized_words.append(interp[1].split(":")[0])
+                last_i = i
+    return lemmatized_words
+
+def count_and_sort_words(words: List[str]) -> List[Tuple[int, str]]:
+    """
+    Returns list of touples (word, occurances_count) sorted by occurances in descending order.
+    """
+    word_count_dict = dict(Counter(words).items())
+    sorted_word_count = [(word, word_count_dict[word]) for word in sorted(word_count_dict, key=word_count_dict.get, reverse=True)]
+    return sorted_word_count
+
+def save_sorted_words(counted_words: List[Tuple[str, int]], file_name: str) -> None:
+    """
+    Saves counted words in csv format into ./output/file_name.csv
+    """
+    # Creating output folder if it does not exist
+    if not os.path.exists("./output"):
+        os.makedirs("./output")
+    # Saving the tokens and their count
+    data = {'word': [word for (word, _) in counted_words], 'count': [count for (_, count) in counted_words]}
+    df = pd.DataFrame(data)
+    df.to_csv(f"./output/{file_name}.csv")
+
+def save_important_phrases(important_phrases: List[str], file_name: str) -> None:
```

Comment:
The same as above - no need for explicit *None* as a return value. Also change typing
```suggestion
def save_important_phrases(important_phrases: list[str], file_name: str):
```